recipe/build_windows.sh

```diff
@@ -0,0 +1,29 @@
+set -ex
+
+export CC=$(which gcc)
+export CXX=$(which g++)
+export NM=$(which nm)
+export AR=$(which ar)
+export LD=$(which ld)
+
+./configure                      \
+    --prefix=${LIBRARY_PREFIX}   \
+    --cc=${CC}                   \
+    --cxx=${CXX}                 \
+    --nm=${NM}                   \
+    --ar=${AR}                   \
+    --ld=${LD}                   \
```

Comment:
Considering
```
+ ./configure '--prefix=D:\bld\ffmpeg_1646622654112\_h_env\Library' --cc= --cxx= --nm= --ar= --ld= --disable-doc --disable-openssl --enable-demuxer=dash --enable-gpl --enable-hardcoded-tables --enable-pic --enable-shared --disable-static --enable-version3 '--x86asmexe=D:\bld\ffmpeg_1646622654112\_build_envLibrarybinnasm.exe' '--pkg-config=D:\bld\ffmpeg_1646622654112\_build_envLibrarymingw-w64binpkg-config.exe'
bash.exe: warning: could not find /tmp, please create! 
[...]
Unknown option "--prefix=%PREFIX%\Library".
```

It seems we're gonna need some quotes around `%PREFIX%\Library`, and set CC etc. differently than with `which`.

Reply:
i think it is the bash slashes.

Reply:
10 min turn around. 

It is different locally, so I can't really test quickly.

Reply:
Well i can't reach azure anyrmore. So feel free to push.. I'm done for the night.